src/xrSound/SoundRender_Source.h

```diff
@@ -1,77 +1,47 @@
 #pragma once
 
-struct OggVorbis_File;
+#include "OggVorbisFile.h"
 
-enum class SoundFormat
+class XRSOUND_API CSoundRender_Source : public CSound_source
```

Comment:
This is internal class, not supposed for export.
```suggestion
class CSoundRender_Source : public CSound_source
```